code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -1,3 +1,4 @@
+#define CALL_BOT_COOLDOWN 900

```

Comment:
It's not meant to be there but it's  a notation for a new line

http://unix.stackexchange.com/questions/32001/what-is-m-and-how-do-i-get-rid-of-it

Pretty neat TBH